inc/notificationtargetticket.class.php

```diff
@@ -39,55 +39,18 @@
 **/
 class NotificationTargetTicket extends NotificationTargetCommonITILObject {
 
-   public $private_profiles = [];
-
    const HEADERTAG = '=-=-=-=';
    const FOOTERTAG = '=_=_=_=';
 
-   /**
-    * @param $entity          (default '')
-    * @param $event           (default '')
-    * @param $object          (default null)
-    * @param $options   array
-    */
-   function __construct($entity = '', $event = '', $object = null, $options = []) {
-      global $CFG_GLPI;
-
-      parent::__construct($entity, $event, $object, $options);
-
-      if (isset($options['followup_id'])) {
-         $this->options['sendprivate'] = $options['is_private'];
-      }
-
-      if (isset($options['task_id'])) {
-         $this->options['sendprivate'] = $options['is_private'];
-      }
-
-   }
-
-
-   /**
-    * @see NotificationTarget::validateSendTo()
-   **/
    function validateSendTo($event, array $infos, $notify_me = false) {
 
       // Always send notification for satisfaction : if send on ticket closure
       // Always send notification for new ticket
-      if (($event != 'satisfaction')
-          && ($event != 'new')) {
-         // Check global ones for notification to myself
-         if (!parent::validateSendTo($event, $infos, $notify_me)) {
-            return false;
-         }
-
-         // Private object and no right to see private items : do not send
-         if ($this->isPrivate()
```

Comment:
Is this one missing? Or implemented another way?

Reply:
This parts has been moved to parent class, as it will be used for Changes and Problems. https://github.com/glpi-project/glpi/pull/4540/files/395de44ee6de273791687bfedc15834e48686b50#diff-ea1db5d4dfd69e74f2c6f17821279f2bR80

Reply:
Ok :)